reports/report_csv.py

```diff
@@ -0,0 +1,174 @@
+import csv, os
+from .report_details import *  # noqa
+from forms.modelutils import (TOPICS, FUNCTION, YESNO, )
+
+def get_gender(gender_id):
+    if gender_id == 1:
+        return "Female"
+    elif gender_id == 2:
+        return "Male"
+    elif gender_id == 3:
+        return "Transgender"
+    else:
+        return "Don't Know"
```

Comment:
We can reuse [`clean_title`](https://github.com/CodeForAfrica/gmmp/blob/dece36517497870f6770a9c84736a17ad0bb52f4/reports/report_builder.py#L81-L89) and `forms/modelutIls.GENDER`